### ISSUE TITLE
The map assumes that GeoJSON cordinates are in wgs84, even when crs is given

### ISSUE THREAD

**Issue by devdattaT** [JasonSanford/geojsonlint.com#8]
The GeoJSON specs mention how the coordinate system can be specified in section 3 of the specs.

The map on the page assumes that the given coordinates are in Lat Long (which is the default, according to the specs) even when the coordinate system is given.

For example, a marker over New York can be given as: 

```
{
    "type": "Point",
    "coordinates": [
       -8238354.76344332,
       4970277.18766739
    ],
    "crs":{"type":"name","properties":{"name":"urn:ogc:def:crs:EPSG::3857"}}
}
```

Now the problem is that the map tries to zoom into this location, assuming that the coordinates are in EPSG4326

I would suggest the following:
*If the crs is specified, then the map should try to project the geometry before displaying.
*If projection is not possible then a warning should be given
*If the coordinates do not fit within the expected range, a warning should be displayed.

**Comment by gabssnake** [comment]
Encountered this same issue as I received an EPSG:27562 crs file and tried to lint/preview it.

Ended up using `reproject` for a quick fix, which uses [`proj4js`](http://proj4js.org) behind the scenes,

**Comment by jtojnar** [comment]
I also tried to use reproject, it is quite ugly but it works:

https://github.com/jtojnar/geojsonlint.com/tree/reproject

**Comment by tyrasd** [comment]
FYI: The new [IETF GeoJSON standard](https://tools.ietf.org/html/rfc7946) doesn't allow custom CRS to be used anymore.

**Comment by Gnafu** [comment]
Then change the title in "Add support for Foreign Member 'crs'"

> However, where all
   involved parties have a prior arrangement, alternative coordinate
   reference systems can be used without risk of data being
   misinterpreted.

**Comment by JasonSanford** [comment]
@tyrasd Good thinking!
